pkgs/by-name/ra/radicle-desktop/package.nix

```
@@ -0,0 +1,127 @@
+{
+  fetchFromGitHub,
+  cargo-tauri,
+  fetchgit,
+  git,
+  glib,
+  gtk3,
+  importNpmLock,
+  lib,
+  libsoup_3,
+  nodejs,
+  openssh,
+  openssl,
+  pkg-config,
+  playwright-driver,
+  radicle-node,
+  rustPlatform,
+  webkitgtk_4_1,
+  wrapGAppsHook4,
+}:
+
+let
+  twemoji-assets = fetchFromGitHub {
+    owner = "twitter";
+    repo = "twemoji";
+    tag = "v14.0.2";
+    hash = "sha256-YoOnZ5uVukzi/6bLi22Y8U5TpplPzB7ji42l+/ys5xI=";
+  };
+in
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "radicle-desktop";
+  version = "0.8.0-unstable-2025-08-15";
+
+  src = fetchgit {
+    url = "https://seed.radicle.xyz/z4D5UCArafTzTQpDZNQRuqswh3ury.git";
+    rev = "e48cc100a519fae72767eda3f4d68104b95a6ccf";
+    hash = "sha256-+Vxw04/+a48DcdXOFG3o/gk8YGwITWEZWhkIJMOfZRA=";
+    leaveDotGit = true;
+    postFetch = ''
+      git -C $out rev-parse HEAD > $out/.git_head
+      git -C $out log -1 --pretty=%ct HEAD > $out/.git_time
+      rm -rf $out/.git
+    '';
+  };
```

Do you mind if we stick with main versioning? The extra/unstable commits seem "irrelevant", so it will be easier/clearer in to bump version on the future

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
  pname = "radicle-desktop";
  version = "0.8.0";

  src = fetchgit {
    url = "https://seed.radicle.xyz/z4D5UCArafTzTQpDZNQRuqswh3ury.git";
    rev = "aeb405aaf53b56a426ab8d68c7f89b8953683224";
    hash = "sha256-+Vxw04/+a48DcdXOFG3o/gk8YGwITWEZWhkIJMOfZRA="; # update this one too
    leaveDotGit = true;
    postFetch = ''
      git -C $out rev-parse HEAD > $out/.git_head
      git -C $out log -1 --pretty=%ct HEAD > $out/.git_time
      rm -rf $out/.git
    '';
  };
```